[UPD] l10n_es_ticketbai_api, l10n_es_ticketbai: Simplificar selección tipo de nº de indentificación

arfbin opened the pull request:
Se simplifica la selección tipo de nº de identificación:

- Eliminar modelo tbai.invoice.customer
- Mejorar la selección del tipo de identificación
- Ajustar la generación de facturas enviadas a Hacienda según la identificación seleccionada

xAdrianC-Kernet commented:
hola @arfbin ,

este parece un cambio grande, he estado revisando un poco y creo que puede tener impacto en modulos que dependen de este, que obviamente tendrán que adaptarse.

Pero me preocupa también que de esta forma, revisando el código solo de vista me da la sensación de que no se va a poder hacer facturas simplificadas sin partner, no? por ejemplo el módulo de tpv sigue funcionando para emitir simplificadas?

arfbin commented:
Hola @xAdrianC-Kernet,

Efectivamente si hubiese otros módulos que dependen de este, habría que adaptarlos si fuera necesario. Si en este caso te refieres al POS, los cambios ya están tenidos en cuenta en el commit: https://github.com/binovo/l10n-spain/commit/94a0d97a88c94994b32cd94ab60aaaf6dd48967e

En este caso el commit está en nuestro fork ya que al no estar disponible todavía el módulo de l10n_ticketbai_pos en OCA no he podido añadir estos cambios, pero sí los hemos tenido en cuenta.

Hay 2 opciones:

- Si se mergea el PR [#2052 ](https://github.com/OCA/l10n-spain/pull/2052) añadiremos el commit sobre este PR
- Si se merge este PR primero, añadiré el commit con los cambios sobre el PR [#2052 ](https://github.com/OCA/l10n-spain/pull/2052)

Espero haberme explicado :)

arfbin committed:
[UPD] l10n_es_ticketbai_api, l10n_es_ticketbai: Simplificar selección tipo de nº de indentificación

- Eliminar modelo tbai.invoice.customer
- Mejorar la selección del tipo de identificación
- Ajustar la generación de facturas enviadas a Hacienda según la identificación seleccionada
